standalone/src/main/scala/filodb.standalone/FiloServer.scala

```diff
@@ -65,6 +68,25 @@ class FiloServer(watcher: Option[ActorRef]) extends FilodbClusterNode {
     val singleton = cluster.clusterSingleton(role, watcher)
     filoHttpServer = new FiloHttpServer(cluster.system, cluster.settings)
     filoHttpServer.start(coordinatorActor, singleton, false, bootstrapper.getAkkaHttpRoute())
+    Thread.sleep(100)
```

Comment:
Why was the sleep required ? The sleep with magic number can create unreliability